app/src/main/admin/components/Touchstones/SingleTouchstoneVersion/ResponsibilitySet.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from "react";
+import {AdminAppState} from "../../../reducers/adminAppReducers";
+import {Dispatch} from "redux";
+import {connect} from "react-redux";
+import {ResponsibilityList} from "./ResponsibilityList";
+import {AnnotatedResponsibilitySet} from "../../../models/AnnotatedResponsibility";
+import {adminTouchstoneActionCreators} from "../../../actions/adminTouchstoneActionCreators";
+import {compose} from "recompose";
+import {ResponsibilitySetWithComments, ResponsibilitySetWithExpectations} from "../../../../shared/models/Generated";
+
+interface ResponsibilitySetProps {
+    responsibilitySet: ResponsibilitySetWithExpectations;
+    canReviewResponsibilities: boolean;
+    responsibilityComments: ResponsibilitySetWithComments[];
+    setCurrentTouchstoneResponsibilitySet: (responsibilitySet: AnnotatedResponsibilitySet) => void;
+}
+
+interface ResponsibilitySetState {
+    annotatedResponsibilitySet: AnnotatedResponsibilitySet;
+}
+
+class ResponsibilitySetComponent extends React.Component<ResponsibilitySetProps, ResponsibilitySetState> {
+
+    constructor(props: ResponsibilitySetProps) {
+        super(props);
+        this.state = {
+            annotatedResponsibilitySet: toAnnotatedResponsibilitySet(props.responsibilitySet, props.responsibilityComments)
+        };
+    }
+
+    handleClick() {
+        this.props.setCurrentTouchstoneResponsibilitySet(this.state.annotatedResponsibilitySet);
+    }
+
+    componentWillReceiveProps(nextProps: Readonly<ResponsibilitySetProps>) {
+        this.setState({
+            annotatedResponsibilitySet: toAnnotatedResponsibilitySet(nextProps.responsibilitySet, nextProps.responsibilityComments)
+        });
+    }
+
+    render(): JSX.Element {
+        const responsibilitySet = this.state.annotatedResponsibilitySet;
+        return <div>
+            <h4>{responsibilitySet.modelling_group_id} (<span>{responsibilitySet.status}</span>)</h4>
+            {this.props.canReviewResponsibilities &&
+            <div style={{marginBottom: "0.5em", display: "flex"}}>
```

Comment:
Really minor thing, but we're tending to use bootstrap classes for margins etc, it's slightly terser

Reply:
Much nicer - thanks!